contrib/kiss-export

```diff
@@ -18,6 +18,11 @@ while read -r file; do
 done < "$KISS_ROOT/var/db/kiss/installed/$pkg/manifest"
 
 # Turn the list of files back into a package.
-tar czf "$pkg#$ver-$rel.tar.gz" -C / -- "$@"
+tar cf - -C / -- "$@" | case ${KISS_COMPRESS:=gz} in
+    bz2) bzip2 -z ;;
+    gz)  gzip -6  ;;
+    xz)  xz -zT 0 ;;
+    zst) zstd -z  ;;
+esac > "$pkg#$ver-$rel.tar.${KISS_COMPRESS:=gz}"
 
-printf 'tarball created in %s\n' "$PWD/$pkg#$ver-$rel.tar.gz"
+printf 'tarball created in %s\n' "$PWD/$pkg#$ver-$rel.tar.${KISS_COMPRESS:=gz}"
```

Comment:
I'm not sure how to fix SC2030-SC2031. Either add shellcheck ignore SC2030-2031, or delete this line (which seems to delete both warnings, but I know the latter is not desirable.)

Reply:
you could actually define it ike this:
A the start you could write this:
`: "${KISS_COMPRESS:=gz}"`
And then you can just use KISS_COMPRESS instead of redefining it with ${KISS_COMPRESS:=gz}

Then shellcheck will be happy